src/urlparse_cmd/process_url.ts

```diff
@@ -168,6 +175,8 @@ export default async function get_metric_scores(filename: string) : Promise<any>
                     url_metrics.license = scores.getLicense();
                     url_metrics.maintainer = await scores.getResponsiveness();
                     url_metrics.correctness = scores.getCorrectness();
+                    url_metrics.pinned_frac = scores.getPinnedDependenciesFraction();
```

Comment:
add await keyword to this line and the line after, both of the called functions return a Promise